spec/system/provider_interface/provider_applications_search_spec.rb

```diff
@@ -143,29 +141,29 @@ def when_i_search_for_candidate_name_with_odd_casing
   end
 
   def then_only_applications_of_that_name_and_status_should_be_visible
-    expect(page).to have_css('.govuk-table__body', text: 'Jim James')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Adam Jones')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Tom Jones')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Bill Bones')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Greg Taft')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Paul Atreides')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Duncan Idaho')
-    expect(page).not_to have_css('.govuk-table__body', text: 'Luke Smith')
+    expect(page).to have_css('.app-application-cards', text: 'Jim James')
+    expect(page).not_to have_css('.app-application-cards', text: 'Adam Jones')
+    expect(page).not_to have_css('.app-application-cards', text: 'Tom Jones')
+    expect(page).not_to have_css('.app-application-cards', text: 'Bill Bones')
+    expect(page).not_to have_css('.app-application-cards', text: 'Greg Taft')
+    expect(page).not_to have_css('.app-application-cards', text: 'Paul Atreides')
+    expect(page).not_to have_css('.app-application-cards', text: 'Duncan Idaho')
+    expect(page).not_to have_css('.app-application-cards', text: 'Luke Smith')
```

Comment:
On reflection, do we need to test for `not_to have_css` of all these things? The affirmative `to have_css` makes sense.

Reply:
Well I guess its actually testing that the names aren't present rather than the classes - will try to think of a neater way to do this.

Reply:
Yeah, I think if we test for the presence of the name we expect, that's good enough for me... I don't think we need the other assertions.

Reply:
👍